ckanext/canada/controller.py

```diff
@@ -110,11 +75,52 @@ def org_key(org):
         )
         return render('organization/index.html')
 
+    def datatable(self, resource_id):
+        echo = int(request.params['sEcho'])
+        search_text = unicode(request.params['sSearch'])
+        offset = int(request.params['iDisplayStart'])
+        limit = int(request.params['iDisplayLength'])
+        sort_cols = int(request.params['iSortingCols'])
+        if sort_cols:
+            sort_by_num = int(request.params['iSortCol_0'])
+            sort_order = ('desc' if request.params['sSortDir_0'] == 'desc'
+                          else 'asc')
```

Comment:
It's a recommendation and a style thing descended from line-based editors, not necessarily part of PEP8. Especially when doing large dicts or lists. Imagine you're using a line based editor on this:

```
x = [
1,
2,
4,
....
]
```

Removing a record is simple and a single keystroke, as you simply delete a line. It's the same reason trailing commas are allowed.